test/json/v1beta1_api_resource_list.json

```diff
@@ -0,0 +1,11 @@
+{
+  "kind": "APIResourceList",
+  "groupVersion": "extensions/v1beta1",
```

Comment:
I'm unsure what do you mean by creating the total v1beta1 list. what I meant is - isn't there a newer (non beta) api version for this?